spinners.json

```diff
@@ -1181,5 +1181,165 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+	"fingerdance": {
+		"interval": 160,
+		"frames": [
+			"☝️",
+			"✌️",
+			"🤘",
+			"🤟",
+			"🖖",
+			"🖐",
+			"✋",
+			"🤚",
+			"👆"
+		]
+	},
+	"fistBump": {
+		"interval": 80,
+		"frames": [
+			"🤜    🤛",
+			"🤜    🤛",
+			"🤜    🤛",
+			" 🤜  🤛",
+			"  🤜🤛",
```

Comment:
The width should be constant.

Reply:
This one is awesome though!

Reply:
Hmmm, I'm really struggling with this one. Is there a unicode character for a space with the width of an emoji?